### DISCUSSION TITLE
out of memory even with multi-card training

### DISCUSSION THREAD

**Discussion by YuanbinLiu** [ACEsuit/mace#120] [Q&A] [answered]
**Describe the bug**
My attempt to train MACE with multiple 4090 cards was unsuccessful. 

**Expected behavior**
I wish to train MACE in parallel using multiple graphics cards.

**Screenshots**![image](https://github.com/ACEsuit/mace/assets/48703058/7d52ef97-4275-42d2-b317-2a23cad399be)

**Comment by davkovacs** [comment]
We are working on multi-GPU training in this PR: https://github.com/ACEsuit/mace/pull/105 
I recommend you either wait a little longer until we merge it, or you can have a go trying it by checking out the fork in the PR. A few people have started testing it and it seems to work.

**Comment by ilyes319** [comment, marked as answer]
Closing this as duplicate of https://github.com/ACEsuit/mace/issues/10, please use the PR #105 for now. For now you can either use a larger GPU or switch to a smaller model/batch size. Reducing the number of channels significantly helps with memory.
